domains/transfer/Transfer.API/API/Shared/Swagger/SwaggerDefaultValues.cs

```diff
@@ -0,0 +1,61 @@
+using System;
+using System.Linq;
+using System.Text.Json;
+using Microsoft.AspNetCore.Mvc.ApiExplorer;
+using Microsoft.OpenApi.Models;
+using Swashbuckle.AspNetCore.SwaggerGen;
+
+namespace API.Shared.Swagger;
+
+public class SwaggerDefaultValues : IOperationFilter
+{
+    /// <inheritdoc />
```

Comment:
We do not really use this elsewhere in the code. All the comments in this class is also a bit different than what we normally do. It looks like something copied from stack overflow or Github Issues.

Reply:
It is a file, taken directly from the setup guide, of the ASP.Versioning WIKI

Reply:
removed comments